doc/conf.py

```diff
@@ -38,7 +38,7 @@
 
 # Used for related links - no need to change
 if 'discourse' in html_context:
-    html_context['discourse_prefix'] = html_context['discourse'] + "/t/"
+    html_context['discourse_prefix'] = "https://discuss.linuxcontainers.org/t/"
```

Comment:
Should we link to the old forum here?

Reply:
Yes, the `discourse_prefix` is used to generate links to specific Discourse topics, and those ones are still on the old forum.
I have a task to update the extension to support different Discourse instances, but for now, all links go to the old one. :)